tests/test_utils.py

```diff
@@ -5,16 +5,15 @@
     AstroDBError,
     ingest_instrument,
 )
+from astrodb_utils.utils import check_in_database
```

Comment:
I notice that in the __init__ you import the various functions from astrodb_utils.utils. Do we want to include this function there as well?